packages/json-deserializer/src/JsonDeserializer.ts

```diff
@@ -0,0 +1,57 @@
+import { PromiseHandler } from "@lambda-middleware/utils";
+import debugFactory, { IDebugger } from "debug";
+import {
+  APIGatewayProxyEvent,
+  APIGatewayProxyResult,
+  Context,
+} from "aws-lambda";
+import { APIGatewayProxyObjectEvent } from "./types/APIGatewayProxyObjectEvent";
+import { RequestBodyNotJsonError } from "./customErrors/RequestBodyNotJsonError";
+
+const logger: IDebugger = debugFactory("@lambda-middleware/json-serializer");
+
+export const jsonDeserializer = <E extends APIGatewayProxyEvent>() => (
+  handler: PromiseHandler<APIGatewayProxyObjectEvent<E>, APIGatewayProxyResult>
+) => async (event: E, context: Context): Promise<APIGatewayProxyResult> => {
+  let body;
+
+  body = deserializeBody(event);
```

Comment:
Could this just be
```suggestion
  const body = deserializeBody(event);
```

Reply:
Hmm yes, I'm not sure why this isn't a const!? I'll fix it shortly.